vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/WindowSteps.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.steps.ui.web;
+
+import org.jbehave.core.annotations.When;
+import org.vividus.monitor.TakeScreenshotOnFailure;
+import org.vividus.selenium.BrowserWindowSize;
+import org.vividus.selenium.IBrowserWindowSizeProvider;
+import org.vividus.selenium.IWebDriverProvider;
+import org.vividus.selenium.manager.IWebDriverManager;
+import org.vividus.softassert.ISoftAssert;
+
+@TakeScreenshotOnFailure
+public class WindowSteps
+{
+    private IWebDriverManager webDriverManager;
+    private IWebDriverProvider webDriverProvider;
+    private IBrowserWindowSizeProvider browserWindowSizeProvider;
+    private ISoftAssert softAssert;
+
+    public WindowSteps(IWebDriverManager webDriverManager, IWebDriverProvider webDriverProvider,
+                       IBrowserWindowSizeProvider browserWindowSizeProvider, ISoftAssert softAssert)
+    {
+        this.webDriverManager = webDriverManager;
+        this.webDriverProvider = webDriverProvider;
+        this.browserWindowSizeProvider = browserWindowSizeProvider;
+        this.softAssert = softAssert;
+    }
+
+    /**
+     * Change current browser window size to the specified one.
+     * @param sizeAsString Any browser window size in pixels, e.g. 800x600.
+     *                    Note: browser window size should be smaller than the current screen resolution.
+     */
+    @When("I resize the current window to `$sizeAsString`")
+    public void resizeCurrentWindow(String sizeAsString)
+    {
+        BrowserWindowSize actualWindowSize = browserWindowSizeProvider
+                .getBrowserWindowSize(webDriverProvider.isRemoteExecution());
+        BrowserWindowSize expectedWindowSize = new BrowserWindowSize(sizeAsString);
```

Comment:
it makes sense to introduce JBehave converter for `BrowserWindowSize`

Reply:
Maybe we could use `org.openqa.selenium.Dimension` instead

Reply:
@valfirst  @uarlouski should we re-use org.openqa.selenium.Dimension for all BrowserWindowSizeProvider methods/introduce to-Dimension converter in scope of this PR?

Reply:
@tshkursk it's up to you